addons/google_calendar/models/calendar_event.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, fields, models
+
+
+class CalendarEvent(models.Model):
+    _inherit = "calendar.event"
+
+    oe_update_date = fields.Datetime(string='Odoo Update Date')
```

Comment:
@atp-odoo if i wrong then correct me but in migration commit no need to change field name, after migration may be reflector the code. :camel: